### DISCUSSION TITLE
Lines appearing in plot() function

### DISCUSSION THREAD

**Discussion by gkrisp98** [TorchIO-project/torchio#884] [Q&A]
Hi, I use the plot() function on nifti files and I got these white lines. Any idea why does this happen and if it affect a classifier somehow ?
![αρχείο λήψης](https://user-images.githubusercontent.com/57015579/169356996-eaaab0f4-9183-4d3f-a2f4-e5b0065ed01b.png)

**Comment by romainVala** [comment]
hello
no those line a just added for visualisation in to of the real data (matplotlib grid function)
it won't affect any training (since it is not in the data)

**Comment by fepegar** [comment]
Hey, @gkrisp98. That happens when `seaborn` is `set` before plotting, as it modifies the Matplotlib parameters so grids are added by default to axes. You can remove the axes or modify Matplotlib/Seaborn settings. As @romainVala said, your image values are not affected by this.
